x/stakeibc/keeper/ibc_handlers.go

```diff
@@ -0,0 +1,131 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+)
+
+// TODO: uncomment below
+// import (
+// 	"encoding/json"
+// 	"fmt"
+
+// 	"github.com/Stride-Labs/stride/x/stakeibc/types"
+// 	sdk "github.com/cosmos/cosmos-sdk/types"
+// 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+// 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+// 	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+// 	"google.golang.org/protobuf/proto"
+
+// 	icatypes "github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/types"
+// )
+
+// Implements core logic for OnAcknowledgementPacket
```

Comment:
Implement OnAcknowledgementPacket logic (TEST-28, TEST-33)

Reply:
does this work right now if we return nil on ack? do we not need to process any info / emit any events?

Reply:
it can just return nil (any processing can be added before returning nil)